internal/spentcalories/spentCalories.go

```diff
@@ -12,7 +16,20 @@ const (
 )
 
 func parseTraining(data string) (int, string, time.Duration, error) {
-	// ваш код ниже
+
+	res := strings.Split(data, ",")
+	if len(res) != 3 {
+		return 0, "", 0, errors.New("количество элементов слайса не равно 3")
+	}
+	steps, err := strconv.Atoi(res[0])
+	if err != nil {
+		return 0, "", 0, errors.New("невозможно преобразовать элемент слайса в int")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```